configstack/stack.go

```diff
@@ -71,8 +71,8 @@ func (stack *Stack) summarizePlanAllErrors(terragruntOptions *options.Terragrunt
 					dependenciesMsg,
 				)
 			}
-		} else if errorStream.Len() > 0 {
-			terragruntOptions.Logger.Printf("Error with plan: %s", output)
+		} else {
+			terragruntOptions.Logger.Println(output)
```

Comment:
It looks like both the if and else block run `terragruntOptions.Logger.Println(output)`. Perhaps extract that out before the if/else and remove the else block entirely?